ydb/_topic_writer/topic_writer_asyncio.py

```diff
@@ -238,15 +279,23 @@ async def write_with_ack_future(
         internal_messages = self._prepare_internal_messages(messages)
         messages_future = [self._loop.create_future() for _ in internal_messages]
 
-        self._messages.extend(internal_messages)
         self._messages_future.extend(messages_future)
 
-        for m in internal_messages:
-            self._new_messages.put_nowait(m)
+        if self._codec == PublicCodec.RAW:
```

Comment:
If need encode message - send it to encode loop, else - direct add to send queue.